app/blueprints/fund/templates/fund_details.html

```diff
@@ -0,0 +1,112 @@
+{% extends "base.html" %}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+{%- from "govuk_frontend_jinja/components/summary-list/macro.html" import govukSummaryList -%}
+{% set showNavigationBar = True %}
+{% set active_item_identifier = "grants" %}
+{% block content %}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        {{ govukBackLink({
+          "text": "Back",
+          "href": url_for("fund_bp.view_all_funds")
+        }) }}
+        <h2 class="govuk-heading-m">{{ fund.name_json["en"] }}</h2>
+        {{ govukSummaryList({
+            "classes": "govuk-!-margin-bottom-9",
+                "rows": [
+                    {
+                        "key": {"text": "Grant name"},
+                        "value": {"text": fund.name_json["en"]},
+                        "actions": {
+                            "items": [
+                                {
+                                    "href": url_for(endpoint="fund_bp.edit_fund", fund_id=fund.fund_id, _anchor="name_en"),
```

Comment:
The anchor tags here need to tie up to form fields on the `FundForm`. This is currently implicit and relies on the IDs being written correctly by a developer in each of these lines.

If we pass a `form` instance into this template, then we can grab the IDs of the fields programmatically which feels like a safer lookup.

Eg

```suggestion
                                    "href": url_for(endpoint="fund_bp.edit_fund", fund_id=fund.fund_id, _anchor=form.name_en.id),
```